### DISCUSSION TITLE
How to add a class to a Block

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#1524] [Questions and Help] [answered]
I want to add class to the block to unify the style of the block. How do I add them?

**Comment by Gervwyk** [comment, marked as answer]
Hi @xiaoxiji1234 

Currently, it is unfortunately not possible, but it makes total sense. We'd like to support this. Will just be some of work to implement. 

In the mean time, you can make use of applying `style` object where you'd like to use, but this can be repetitive unfortunately. By using a `_ref` you can at least make sure you standardize how you apply these styles.

**Comment by xiaoxiji1234** [comment]
I got it, Thank you

**Reply by drewlsvern** [reply]
@Gervwyk Is there an ETA when this might be added?

**Reply by Gervwyk** [reply]
Hi @drewlsvern. Unfortunately, not atm. Perhaps provide an example of your use case, maybe I can come up with a workaround for you in the mean time?
